lib/tweet_item_model.dart

```diff
@@ -0,0 +1,8 @@
+class TweetItemModel {
```

Comment:
to keep app simple, I would move that to `tweet_helper.dart` file. Don't be afraid to put multiple classes into a single file if they are related/used together, the habbit of using separate file for each class came from Java/C#/etc. and is at least controversial